src/index.js

```diff
@@ -56,6 +58,10 @@ const _init = (options, readyCallback) => {
     window._fs_is_outer_script = true;
   }
 
+  if (options.debug && !options.script) {
+    options.script = 'edge.fullstory.com/s/fs-debug.js';
```

Comment:
This is interesting; would we want to preserve the script value if `debug` is set? Or is API behavior clearer if we _always_ set the script to point to `fs-debug.js` when the `options.debug === true`, regardless of the script value passed into `init()`?

=

Reply:
This snippet change put the existing SDK API in an awkward state. The assumption I'm working under here is that if someone is overriding the default value of `script`, it's because they're not pointing it at FullStory's server, so it doesn't seem correct to `s/fs.js$/fs-debug.js/` on that value and assume the resulting script will exist.

Having slept on this, I think perhaps tweaking this conditional so that if `options.debug` is set *and* `options.script` is set, we log a warning so that the `debug: true` is at least not silently ignored.

Reply:
Do you know what the behavior of the browser API (not wrapped in SDK) is?

i.e. if 

```
window['fs_source'] = '{some path, maybe not hosted by FS}';
window['fs_debug'] = true;
```

What happens?

Reply:
@luser SGTM, go ahead and make the change and I'll approve.